gradle/publish.gradle

```diff
@@ -32,20 +30,26 @@ publishing {
     }
 
     repositories {
-        if ("${project.version}".endsWith('-SNAPSHOT')) {
-            sonatypeSnapshots {
-                url = "https://s01.oss.sonatype.org/content/repositories/snapshots/"
-                credentials {
-                    username = project.findProperty('sonatype_username')
-                    password = project.findProperty('sonatype_password')
-                }
-            }
-        } else {
-            sonatypeMavenCentral {
-                url = "https://s01.oss.sonatype.org/service/local/"
-                credentials {
-                    username = project.findProperty('sonatype_username')
-                    password = project.findProperty('sonatype_password')
+        maven {
+            url = layout.buildDirectory.dir('staging-deploy')
+        }
+    }
+}
+
+jreleaser {
+    signing {
+        active = 'ALWAYS' // both 'RELEASE' and 'SNAPSHOT' enabled
+        armored = true
+        verify = false // disables need for 'JRELEASER_GPG_PUBLIC_KEY'
+    }
+    deploy {
+        maven {
+            mavenCentral {
+                sonatype {
+                    active = 'ALWAYS' // both 'RELEASE' and 'SNAPSHOT' enabled
+                    url = 'https://central.sonatype.com/api/v1/publisher'
+                    applyMavenCentralRules = true
+                    stagingRepository(layout.buildDirectory.dir('staging-deploy'))
                 }
```

Comment:
I *think* `namespace` has to be set as well.

It default to the groupId, but that doesn't match the publication namespace exactly. (The groupId here is `com.contentgrid.hateoas`, set in https://github.com/xenit-eu/contentgrid-hateoas/blob/main/gradle.properties#L1